src/layouts/PLayoutFull/PLayoutFull.vue

```diff
@@ -12,7 +12,6 @@
 .p-layout-full { @apply
   relative
   w-full
-  h-full
```

Comment:
Not sure if we even use this one anymore but are you sure about removing this one? This one is for full screen stuff and h-full kinda makes sense here.

Reply:
I couldn't actually find an example of the full layout in the app, so I just undid this piece to be safe. 👍